config/i18n.json

```diff
@@ -811,7 +811,7 @@
     "Unequipped": "Unequipped",
     "VendorsCannotEquip": "You don't have these items. Tap to pick a replacement or click the X to remove:",
     "WeaponsOnly": "Weapons Only",
-    "NotStarted": "Waiting for other actions",
+    "NotStarted": "Waiting for other actions or profile loading to finish",
```

Comment:
```suggestion
    "NotStarted": "Waiting for other actions to complete, or an inventory refresh to finish loading",
```
maybe?

Reply:
inserts more of a "hang may be on bungie's end" feeling than a "dim has decided not to yet"